### PR TITLE
doc: add missing write callback to example

### PR DESCRIPTION
**Pull request by bonifaido**

### CODE REVIEW

File: docs/src/stream.rst [hunk 1]
```diff
@@ -148,6 +148,10 @@ API
 
     ::
 
+        void cb(uv_write_t* req, int status) {
+            // Logic which handles the write result
```
Comment (saghul) [inline docs/src/stream.rst]: Please use C style comments: /* ... */

**Comment by saghul** [comment]
Can you please adjust the commit log following the guidelines in CONTRIBUTING.md ? Thanks!

**Comment by bonifaido** [comment]
Please check if these changes adhere to the style.

**Comment by saghul** [comment]
Yep, LGTM!

**Comment by saghul** [comment]
Landed in 729d685, thanks!

PS: I see this is your first code contribution to libuv, welcome onboard! :-)

**Comment by bonifaido** [comment]
You're welcome and thank you! :-)

**Pull request closed without merging**